HappyAnding/HappyAnding/ViewModel/ShortcutsZipViewModel.swift

```diff
@@ -940,3 +942,75 @@ extension ShortcutsZipViewModel {
         setData(model: comments)
     }
 }
+
+//MARK: - 등급 관련 함수
+extension ShortcutsZipViewModel {
+    
+    func checkShortcutGrade(userID: String?) -> ShortcutGrade {
+        var count = 0
+        
+        if let userID {
+            count = allShortcuts.filter { $0.author == userID }.count
+        } else {
+            count = shortcutsMadeByUser.count
+        }
+        
+        if count < 1 {
+            return ShortcutGrade.level0
+        } else if count >= 1 && count < 5 {
+            return ShortcutGrade.level1
+        } else if count >= 5 && count < 10 {
+            return ShortcutGrade.level5
+        } else if count >= 10 && count < 25 {
+            return ShortcutGrade.level10
+        } else if count >= 25 && count < 50 {
+            return ShortcutGrade.level25
+        } else {
+            return ShortcutGrade.level50
+        }
```

Comment:
count 변수에 대한 조건문이니까 if - else if - else보다 switch 문으로 표현하는건 어떨까요?
리턴형이 있으니 멤버를 생략하는게 더 좋아보이네요

```suggestion
        switch count {
        case ..<1:
            return .level0
        case 1..<5:
            return .level1
        case 5..<10:
            return .level5
        case 10..<25:
            return .level10
        case 25..<50:
            return .level25
        default:
            return .level50
        }
```